configs/disco_diffusion/tutorials.ipynb

```diff
@@ -0,0 +1,1443 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Disco Diffusion is a colab work cooperated by many talented people. From the algorithm aspect, this work is based on guided-diffusion and clip-guided diffusion. \n",
+    "\n",
+    "The generation process of disco-diffusion consists of following steps,\n",
+    "1. Disco-diffusion takes as inputs user-provided text prompts and image prompts.\n",
+    "2. A pretrained clip model calculates the embeddings of the text and image prompts.\n",
+    "3. The Unet will predict the original sample.\n",
+    "3. A list of image cutouts will be generated from original sample by a cutter.\n",
+    "4. The generation process is optimized by calculating the similarity between the embeddings of the image cutouts, text and image prompts, and other losses.\n",
+    "5. Finally, with the carefully-designed loss functions, the model enables classifier-guidance samping to generate desired images according to the text and image prompts.\n",
+    "\n",
+    "So corresponding to above processes, we will introduce what to set for generating and show your results by adjusting arguments/ configs. \n",
+    "\n",
+    "The contents of this tutorials are as follows:\n",
+    "\n",
+    "[1.Runtime Settings](#1-Runtime-Settings)\n",
+    "\n",
+    "[2.Unet Settings](#2-Unet-Settings)\n",
+    "\n",
+    "[3.CLIP Models Settings](#3-CLIP-Models-Settings)\n",
+    "\n",
+    "[4.Diffusion Scheduler Settings](#4-Diffusion-Scheduler-Settings)\n",
+    "\n",
+    "[5.Loss Settings](#5-Loss-Settings)\n",
+    "\n",
+    "[6.Cutter Settings](#6-Cutter-Settings)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install MMEditing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check PyTorch version\n",
+    "!pip3 list | grep torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install mmcv dependency via openmim\n",
+    "!pip3 install openmim\n",
+    "!mim install 'mmcv>=2.0.0rc1'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install mmediting from source\n",
+    "%cd ../..\n",
+    "!pip3 install -e .\n",
+    "%cd configs/disco_diffusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Runtime Settings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "import time  # 引入time模块\n",
+    "from mmengine import Config, MODELS\n",
+    "from mmedit.utils import register_all_modules\n",
+    "from mmedit.registry import MODULES\n",
+    "\n",
+    "from torchvision.transforms import ToPILImage, Normalize, Compose\n",
+    "from IPython.display import Image\n",
```

Comment:
Add `from mmcv import tensor2imgs` and `from matplotlib import pyplot as plt` here.